Git-Squash.md

```diff
@@ -0,0 +1,22 @@
+# Squashing Multiple Commits Into One
+
+This is an awesome feature of `rebase` that can be used in the `interactive` mode. To squash the last _n_ commits into one, run the following command:
+
+```
+git rebase -i HEAD~n
+```
+
+That will open up a text-editor with something similar to the following:
+
+```
```

Comment:
I didn't mean syntax highlight for that although those should also be properly formatted :grin: 
```text
    ```sh
    $ git rebase -i HEAD~n
    ```
```

I meant for these 
```
    ```text
    pick commit_1
    pick commit_2
    ```
```